libs/external-db-mysql/src/mysql_operations.ts

```diff
@@ -0,0 +1,17 @@
+import { IDatabaseOperations } from '@wix-velo/velo-external-db-types'
+import { promisify } from 'util'
+import { notThrowingTranslateErrorCodes } from './sql_exception_translator'
+
+export default class DatabaseOperations implements IDatabaseOperations {
+    pool: any
+    query: any
```

Comment:
maybe `MySqlQuery` type rather than `any`?

Reply:
Because of the promisify that's not the type